core/xml.js

```diff
@@ -365,7 +365,14 @@ Blockly.Xml.textToDom = function(text) {
   var doc = Blockly.utils.xml.textToDomDocument(text);
   if (!doc || !doc.documentElement ||
       doc.getElementsByTagName('parsererror').length) {
-    throw Error('textToDom was unable to parse: ' + text);
+    var oParser = new DOMParser();
+    // Attempt to parse as html to deserialize control chars.
+    doc = oParser.parseFromString(text, 'text/html');
+    if (!doc || !doc.body.firstChild ||
+        doc.body.firstChild.nodeName.toLowerCase() != 'xml') {
+      throw Error('textToDom was unable to parse: ' + text);
```

Comment:
I think this message is out of date.  `textToDom` should probably be `DOMParser`.